MingW broken?

It looks like b0503d282ede2a6a91eb9e0f97a3d5c8e3147017 broke MingW on the builders...it looks like it's looking for FluidSynth 2.2.0, finds 2.4.7, and panics.

```
CMake Error at D:/a/_temp/msys64/mingw64/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:227 (message):
  Could NOT find FluidSynth (missing: FluidSynth_LIBRARY) (found suitable
  version "2.4.7", minimum required is "2.2.0")
```

https://github.com/libsdl-org/SDL_mixer/actions/runs/17447021195/job/49543866718

--- a/cmake/FindFluidSynth.cmake
+++ b/cmake/FindFluidSynth.cmake
@@ -4,7 +4,7 @@ find_package(PkgConfig QUIET)
 pkg_check_modules(PC_FLUIDSYNTH QUIET fluidsynth)
 
 find_library(FluidSynth_LIBRARY
-    NAMES fluidsynth libfluidsynth
+    NAMES fluidsynth fluidsynth-3 libfluidsynth
     HINTS ${PC_FLUIDSYNTH_LIBDIR}
 )
 

